tool/tctl/common/status_command.go

```diff
@@ -101,6 +101,15 @@ func (c *StatusCommand) Status(client auth.ClientI) error {
 			}
 			info := fmt.Sprintf("%v CA ", strings.Title(string(ca.GetType())))
 			rotation := ca.GetRotation()
+			if rotation.Phase == types.RotationPhaseStandby && len(ca.GetAdditionalTrustedKeys().SSH) > 0 {
```

Comment:
```suggestion
			if rotation.Phase == types.RotationPhaseStandby && len(ca.GetAdditionalTrustedKeys().SSH) > 0 && ca.GetAdditionalTrustedKeys().SSH[0].PrivateKeyType == types.PrivateKeyType_PKCS11 {
```

Reply:
Without this restriction I'm catching the symmetric case where a non-HSM auth is added to a cluster with no Active raw keys